roles/nginxplus/tasks/plus/setup-debian.yml

```diff
@@ -1,17 +1,17 @@
 ---
-- name: "Install: Add NGINX Plus Repository"
-  apt_repository:
-    repo: deb https://plus-pkgs.nginx.com/{{ ansible_distribution|lower }} {{ ansible_distribution_release }} nginx-plus
+- name: nginxplus | Add NGINX Plus Repository
+  ansible.builtin.apt_repository:
+    repo: deb {{ nginx_signed_statement }} https://pkgs.nginx.com/plus/ubuntu {{ ansible_distribution_release }} nginx-plus
     filename: nginx-plus
     update_cache: false
 
-- name: "Install: Verify NGINX Plus License"
-  blockinfile:
+- name: nginxplus | Verify NGINX Plus License
+  ansible.builtin.blockinfile:
```

Comment:
This is the second `blockinfile` task for the `/etc/apt/apt.conf.d/90nginx` file. Could we use a template instead?